tests/contrib/idempotent/test_idempotent_runner.py

```diff
@@ -493,13 +529,17 @@ def test_mds_change(self, txt_output_data_set):
 class TestSeqentialRunnerBranchlessPipeline:
     def test_no_input_seq(self, branchless_no_input_pipeline):
         outputs = IdempotentSequentialRunner().run(
-            branchless_no_input_pipeline, DataCatalog()
+            branchless_no_input_pipeline, DataCatalog({
+                IDEMPOTENT_STATE_STORAGE_CATALOG_NAME: MemoryDataSet(data={}),
```

Comment:
This is too much boiler plate. Instead of having to add an `IDEMPOTENT_STATE_STORAGE_CATALOG_NAME` catalog everywhere, make the `IdempotentSequentialRunner` easier to test.

Reply:
You can do this by setting `state_store` in the `__init__` and loading the state data from the catalog in a different way, like a `.load` function, so you can instantiate the StateStorage, and load the `catalog` later.